### DISCUSSION TITLE
This new Feed is Garbage

### DISCUSSION THREAD

**Discussion by TanmoySG** [community/community#37461] [Other Feature Feedback, Questions, & Ideas]
How can the previous feed be enabled? Add the tabbed approach if you are so eager to push random activities to user's feed. Bring back the old Explore Page.

**Comment by queenofcorgis** [comment]
Thanks for the feedback! Could you provide more details around what you like about the previous Feed?

**Reply by TanmoySG** [reply]
- The activity from people I follow are missing, that was really helpful
- The Releases from repos I follow/star are missing, that was really helpful
- There are bunch of stuff that I dont need in the current feed, like random user/orgs and their activities
- The current feed is mostly filled with random stuff that I dont follow and mostly are not useful.
- I liked everything about the previous feed. The tabbed approach of creating a clear divide between new unknown stuff and activities from followed people/orgs made much more sense than this disaster in form of current feed.

**Reply by Rillieux** [reply]
I just want ot add I agree fully with the above.
